offlineimap/repository/Base.py

```diff
@@ -213,7 +219,7 @@ def sync_folder_structure(self, dst_repo, status_repo):
                 # Don't create missing folder on readonly repo.
                 break
 
-            if dst_folder.sync_this and not dst_name_t in src_folders:
+            if dst_folder.sync_this and not dst_name_t in src_list:
```

Comment:
The goal is to have this:

Name -> UTF7 + nametrans -> List
Name -> UTF7 + nametrans + separator -> hash
If we don't use this we'll compare things like that:
IMAP(utf7) != Local folder (utf8) even if the IMAP folder is in fact the same as the utf8, not doing this simply makes the decodedfoldersname feature not usable.